common/performance-timeline-utils.js

```diff
@@ -10,7 +10,8 @@ function wp_test(func, msg, properties)
     if (performanceNamespace === undefined || performanceNamespace == null)
     {
       // show a single error that window.performance is undefined
-      test(function() { assert_true(performanceNamespace !== undefined && performanceNamespace != null, "window.performance is defined and not null"); }, "window.performance is defined and not null.", {author:"W3C http://www.w3.org/",help:"http://www.w3.org/TR/navigation-timing/#sec-window.performance-attribute",assert:"The window.performance attribute provides a hosting area for performance related attributes. "});
+      // The window.performance attribute provides a hosting area for performance related attributes.
+      test(function() { assert_true(performanceNamespace !== undefined && performanceNamespace != null, "window.performance is defined and not null"); }, "window.performance is defined and not null.", {author:"W3C http://www.w3.org/",help:"http://www.w3.org/TR/navigation-timing/#sec-window.performance-attribute"});
```

Comment:
For `help` and `author` may I suggest moving the information to the top of the test files using the existing style for that? Otherwise these same tests will need to be updated again to remove the properties entirely.

Reply:
@foolip , I move them to the head of test files.